app/controllers/admin/news_controller.rb

```diff
@@ -0,0 +1,53 @@
+# encoding: utf-8
+class Admin::NewsController < ApplicationController
+  before_action :authorize
+
+  def index
+    @news_grid = initialize_grid(News, include: :user,
+                                       order: 'news.created_at',
+                                       order_direction: :desc)
+  end
+
+  def new
+    @news = News.new
+  end
+
+  def edit
+    @news = News.find(params[:id])
+  end
+
+  def create
+    @news = News.new(news_params)
+    @news.user = current_user
+    if @news.save!
+      redirect_to news_path(@news), notice: alert_create(News)
+    else
+      render :new
+    end
+  end
+
+  def update
+    @news = News.find(params[:id])
+    if @news.update!(news_params)
+      redirect_to news_path(@news), notice: alert_update(News)
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @news = News.find(params[:id])
+    @news.destroy!
+    redirect_to admin_news_index_path, notice: alert_destroy(News)
+  end
+
+  private
+
+  def authorize
```

Comment:
The problem is the namespace.

This could solve it, wonder if it is worth it, might be hard to add with our `Permission`-setup.
https://github.com/ryanb/cancan/wiki/Authorization-for-Namespaced-Controllers